codespell_lib/data/dictionary.txt

```diff
@@ -18527,10 +18534,13 @@ provate->private, provide,
 provded->provided
 provder->provider
 provdided->provided
+provdidet->provided, provident,
 provdie->provide
 provdied->provided
 provdies->provides
 provding->providing
+proveable->provable
```

Comment:
Remove this and put a note in the relevant issue, it's British English:
https://www.collinsdictionary.com/dictionary/english/proveable

Reply:
Done